static/js/protein/chart.ts

```diff
@@ -465,8 +590,16 @@ export function drawDiseaseGeneAssocChart(
   if (_.isEmpty(data)) {
     return;
   }
+  //Finds the length of the object array
+  const arrayLength = Object.keys(data).length;
+  let height = null;
```

Comment:
you don't actually have to set height as anything, you can do `let height: number;`

Reply:
Done!